projects/vdk-plugins/vdk-kerberos-auth/tests/test_kerberos.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import unittest
+
+from click.testing import Result
+from vdk.plugin.kerberos import kerberos_plugin
+from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
+from vdk.plugin.test_utils.util_funcs import jobs_path_from_caller_directory
+
+
+class TemplateRegressionTests(unittest.TestCase):
+    def setUp(self) -> None:
+        self.__runner = CliEntryBasedTestRunner(kerberos_plugin)
+
+    def test_no_authentication(self):
```

Comment:
Did you search for some mock/stub kerberos server we can use (similar to how we use https://pypi.org/project/pytest-httpserver/ for HTTP mock). I did quick google and did not see something but maybe you can find it ...

Reply:
So far I haven't found anything either.